app/assets/javascripts/simulations.js

```diff
@@ -0,0 +1,20 @@
+// # Place all the behaviors and hooks related to the matching controller here.
+// # All this logic will automatically be available in application.js.
+// # You can use CoffeeScript in this file: http://coffeescript.org/
+
+$(document).ready(function() {
```

Comment:
@DerekStride It would be less wasteful to use `document.getElementById('simulation-table')` Since we're just trying to get a DOM object, not a JQuery object, there's no need to use the selector engine and object wrapper in this case